Source: https://github.com/hyperion-project/hyperion/commit/ad764cfeabfdea879a3cef92e291e5fc3888fc2f
Upstream: Yes
Reason: FTBFS


commit ad764cfeabfdea879a3cef92e291e5fc3888fc2f
Author: jelle van der Waa <jelle@vdwaa.nl>
Date:   Sat Oct 21 23:44:08 2017 +0200

    Fix no decleration of `assert` with GCC 7.2 (#767)
    
    Add a missing include for cassert to fix the -fpermissive compiler
    error.

diff --git include/utils/Image.h include/utils/Image.h
index e8c82e4..dad57a2 100644
--- include/utils/Image.h
+++ include/utils/Image.h
@@ -1,6 +1,7 @@
 #pragma once
 
 // STL includes
+#include <cassert>
 #include <vector>
 #include <cstdint>
 #include <cstring>
